database/config.go

```diff
@@ -202,6 +272,28 @@ func (t *tableConfigStore) Delete(tableName string) error {
 	return err
 }
 
+func (t *tableInfoStore) ListTables() ([]string, error) {
+	it := t.st.NewIterator(engine.IteratorConfig{Reverse: false})
```

Comment:
🌷 It's reverse by default
```suggestion
	it := t.st.NewIterator(engine.IteratorConfig{})
```